packages/flame-tokens/README.md

```diff
@@ -57,41 +57,45 @@ Note that this command will be run automatically when we publish to npm.
 ### Import SCSS variables
 
 ```scss
-@import '@lightspeed/flame-tokens/index.scss';
+@import '@lightspeed/flame-tokens/sass/index.scss';
```

Comment:
Isn't this supposed to be `@import '@lightspeed/flame-tokens/sass/flame/index.scss';` based on your changes?